conformance/conformance_test.go

```diff
@@ -0,0 +1,265 @@
+package conformance
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/runfiles"
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common"
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/google/cel-go/ext"
+	"github.com/google/cel-spec/proto/test/v1/testpb"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/encoding/prototext"
+	"google.golang.org/protobuf/testing/protocmp"
+
+	test2pb "github.com/google/cel-spec/proto/test/v1/proto2/test_all_types"
+	test3pb "github.com/google/cel-spec/proto/test/v1/proto3/test_all_types"
+	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
+)
+
+type testsFlag []string
+
+func (t *testsFlag) String() string {
+	return strings.Join(([]string)(*t), ",")
+}
+
+func (t *testsFlag) Set(v string) error {
+	*t = strings.Split(v, ",")
+	for i, v := range *t {
+		(*t)[i] = strings.TrimSpace(v)
+	}
+	return nil
+}
+
+func (t *testsFlag) Get() any {
+	return ([]string)(*t)
+}
+
+type skipTestsFlag []string
+
+func (t *skipTestsFlag) String() string {
+	return strings.Join(([]string)(*t), ",")
+}
+
+func (t *skipTestsFlag) Set(v string) error {
+	*t = strings.Split(v, ",")
+	for i, v := range *t {
+		(*t)[i] = strings.TrimSpace(v)
+	}
+	return nil
+}
+
+func (t *skipTestsFlag) Get() any {
+	return ([]string)(*t)
+}
+
+var (
+	dashboard bool
+	tests     testsFlag
+	skipTests skipTestsFlag
+
+	envWithMacros *cel.Env
+	envNoMacros   *cel.Env
+)
+
+func init() {
+	flag.BoolVar(&dashboard, "dashboard", false, "Dashboard.")
+	flag.Var(&tests, "tests", "Paths to run, separate by a comma.")
```

Comment:
Could you document an example path?

Reply:
This tool is never meant to be invoked directly, only through the macros. Bazel passes through the paths in a way that `github.com/bazelbuild/rules_go/go/runfiles` understands. Which is a little programmatic.